e2e/autoscript-suites/README.md

```diff
@@ -0,0 +1,58 @@
+# Autoscript Suites
+
+This is a package that contains [playwright](https://playwright.dev/)
+tests for the [autoscript-apps](../autoscript-apps) application.
+
+## How to write a test
+
+In order to write a test navigate to the [src](./src) directory.
+
+Once there you will see a folder called suites.
+In here we can write a test which follows the name of the
+feature we are testing, and a .test.ts suffix.
+
+`my-auth.test.ts` as an example.
+
+Next, we typically use `playwrights` `test` blocks and leverage a
+helper file called [setupAndGo](./src/utilities).
+
+Setup and go will do most of the heavy lifting, and launch the browser,
+and setup the url state so that the proper values passed through.
+
+Note: Remember, setup and go is a promise returning function
+so please use an async callback in your `test` block.
+
+Furthermore, setup and go will only output console logs to the terminal
```

Comment:
Does `setupAndGo` do any logging to the Chrome browser? It listens to the logs and appends it to an array, but it doesn't specifically log anything of value, yeah?